11-payment-encoding.md

```diff
@@ -253,6 +268,19 @@ engines that support SQL or other dynamically interpreted querying languages.
 
 Don't be like the school of [Little Bobby Tables](https://xkcd.com/327/).
 
+## Feature Bits
+
+Feature bits allow forward and backward compatibility, and follow the
+_it's ok to be odd_ rule.
+
+The field is big-endian.  The least-significant bit is numbered 0,
+which is _even_, and the next most significant bit is numbered 1,
+which is _odd_.
+
+| Bits | Name        |Description            | Link                                           |
+|------|-------------|-----------------------|------------------------------------------------|
+| 0/1  | `basic_mpp` | Payee understands M.P.P.| [BOLT #4](04-onion-routing.md#basic-multi-part-payments) |
```

Comment:
Will there also be a global feature bit that advertises mpp in the graph, so that spontaneous AMP payments can be made too?

I think that in that case the invoice flag is still required, because the payer may not have the (up to date) node features of the payee? Does this mean that `basic_mpp` implies `var_onion_optin`?

Reply:
Yes, and yes.  I'll add that specifically.